src/features/todo.js

```diff
@@ -0,0 +1,85 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const loadLocalStorage = () => {
+    return window.localStorage.getItem("todos")
```

Comment:
what is the reason you are using Windows to access local storage? if there is no reason you can only use localStorage like this 
 ```suggestion
    return localStorage.getItem("todos")
```